proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -0,0 +1,328 @@
+}
+
+message ListConnectorsRequest {
+  repeated string expand = 1 [
+    (buf.validate.field).repeated.max_items = 2,
+    (buf.validate.field).repeated.unique = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {description: "The expand parameter determines the details of the connector response, it can be of type info, status or a combination of both"}
+  ];
+  string cluster_name = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+
+  option (buf.validate.message).cel = {
+    id: "expand_can_only_be_status_or_info",
+    message: "Valid expand values are: info, status",
+    expression: "this.expand.all(x, x in [\"info\", \"status\"])"
+  };
+
+  // Value of the next_page_token field returned by the previous response. If not provided, the system assumes the first page is requested.
+  string page_token = 3;
+}
+
+message RestartConnectorRequest {
+  message Options {
+    bool include_tasks = 1;
+    bool only_failed = 2;
+  }
+  string cluster_name = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+  string name = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+
+  Options options = 3;
+}
+
+message DeleteConnectorRequest {
+  string cluster_name = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+  string name = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+}
+
+message PauseConnectorRequest {
+  string cluster_name = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+  string name = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+}
+
+message ResumeConnectorRequest {
+  string cluster_name = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+  string name = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+}
+
+message GetConnectorRequest {
+  string cluster_name = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+  string name = 2 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+}
+
+message CreateConnectorRequest {
+  string cluster_name = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true
+  ];
+  ConnectorSpec connector = 2;
+}
+
+message GetConnectorResponse {
+  ConnectorSpec connector = 1;
+  HolisticState holistic_state = 4;
+  repeated ConnectorError errors = 5;
+}
+
+message CreateConnectorResponse {
+  string name = 1;
+  // cod fig is the connector cron configuration map, every key is a string
+  map<string, string> config = 2;
+  repeated TaskInfo tasks = 3 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
Aren't all of these fields in the response schemas OUTPUT_ONLY? We do not share structs for create/responses, so I'm wondering whether we still need the OUTPUT_ONLY annotation in the response structs